pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/event/publish/pubsub/PubSubPublisherInitializer.java

```diff
@@ -58,7 +59,11 @@ public static void shutdownPublishers() {
             try {
                 publisher.getValue().shutdown();
             } catch (Exception e) {
-                log.error("Error shutting down publisher {}", publisher.getKey(), e);
+                if (!SparkBootUtil.isShuttingDown()) {
+                    log.error("Error shutting down publisher {}", publisher.getKey(), e);
+                } else {
+                    log.info("Exception during shutdown", e);
```

Comment:
same here, I am just wondering if logging an exception at info level will make it easy to miss?